lib/utils/findMediaFeatureNamesSearch.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const { TokenType, isToken, stringify, tokenize } = require('@csstools/css-tokenizer');
+const {
+	isTokenNode,
+	parseCommaSeparatedListOfComponentValues,
+	isSimpleBlockNode,
+} = require('@csstools/css-parser-algorithms');
+const {
+	isGeneralEnclosed,
+	isMediaFeature,
+	isMediaQueryInvalid,
+	parseFromTokens,
+} = require('@csstools/media-query-list-parser');
+
+/** @typedef {Array<import('@csstools/media-query-list-parser').MediaQuery>} MediaQueryList */
+/** @typedef {import('@csstools/css-tokenizer').TokenIdent} TokenIdent */
+/** @typedef {{ stringify: () => string }} MediaQuerySerializer */
+
+const rangeFeatureOperator = /[<>=]/;
+
+/**
+ * Search a CSS string for Media Feature names.
+ * For every found name, invoke the callback, passing the token
+ * as an argument.
+ *
+ * Found tokens are mutable and modifications made to them will be reflected in the output.
+ *
+ * This function supports some non-standard syntaxes like SCSS variables and interpolation.
+ *
+ * @param {string} mediaQueryParams
+ * @param {(mediaFeatureName: TokenIdent) => void} callback
+ *
+ * @returns {MediaQuerySerializer}
+ */
+module.exports = function findMediaFeatureNamesSearch(mediaQueryParams, callback) {
```

Comment:
[suggest] I feel that `find` and `Search` are duplicated. How about this? ⬇️ 

```suggestion
module.exports = function findMediaFeatureNames(mediaQueryParams, callback) {
```

Reply:
Sounds good! Thank you